penny_university_frontend/src/tests/components/Alerts.test.tsx

```diff
@@ -5,22 +5,25 @@ import { ErrorAlert } from '../../components/alerts'
 
 describe('error alert component', () => {
   it('renders correctly', () => {
-    const component = shallow(<ErrorAlert error="error" dismiss={jest.fn()} />)
+    const error = {error: "error"}
+    const component = shallow(<ErrorAlert error={error} dismiss={jest.fn()} />)
     expect(component).toMatchSnapshot()
```

Comment:
cool, hadn't noticed that this was a thing. How are snapshots updated?

Reply:
They are updated if you delete them, and then run the test again.